CvGameCoreDLL/CvCityAI.cpp

```diff
@@ -4052,6 +4052,12 @@ int CvCityAI::AI_buildingValueThreshold(BuildingTypes eBuilding, int iFocusFlags
 						{
 							iTempValue += ((kBuilding.getBonusYieldModifier(iJ, iI) * getBaseYieldRate((YieldTypes)iI)) / 12);
 						}
+
+						// 1SDAN
+						if (kBuilding.getBonusYieldChange(iJ, iI) > 0)
+						{
+							iTempValue += (kBuilding.getBonusYieldChange(iJ, iI) * countNumBonusPlots() * getBaseYieldRate((YieldTypes)iI) / 6);

```

Comment:
I think this values the yield change too lowly. Compare above `kBuilding.getYieldChange(iI) * 6` for the flat yield increase from a building.

Reply:
I'm gonna be honest, I didn't know what to put there so I just kind of guessed based on the nearby values. What do you think I should put?

Also, how do I remove the DLL from the pull request?

Reply:
Yeah, I also never know what to use to I just try to keep it consistent with what's already there. I see no reason not to weigh yield from improvements the same way as yields from buildings, so `*6` should be the right factor.

If you're using Tortoise, it has a "delete (keep local)" option that should do the right thing.